test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -38,6 +38,27 @@ segmentItems:
         parentSpanId: 1, parentTraceSegmentId: not null, parentServiceInstance: not
           null, parentService: not null, traceId: not null}
       skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/subscribeOn/elastic
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 67
+      isError: false
+      spanType: Entry
+      peer: ''
+      tags:
+      - {key: url, value: 'http://localhost:18080/testcase/subscribeOn/elastic'}
+      - {key: http.method, value: GET}
+      - {key: http.status_code, value: '200'}
+      refs:
+      - {parentEndpoint: GET:/projectA/testcase, networkAddress: 'localhost:18080', refType: CrossProcess,
```

Comment:
2 things.
1. About test output, I mean following [this doc](https://skywalking.apache.org/docs/skywalking-java/v8.9.0/en/setup/service-agent/java-agent/plugin-test/), run plugin tests locally, and verify this expectedData.yaml.
2. This is a new entry span, where is the exit span? I am confused about this.